flink-test-utils-parent/flink-connector-testing/src/main/java/org/apache/flink/connectors/test/common/environment/MiniClusterTestEnvironment.java

```diff
@@ -103,6 +113,9 @@ public void startUp() throws Exception {
             return;
         }
         this.miniCluster.before();
+        this.metricReporter =
```

Comment:
I'm sorry but how does this even work? Where are we configuring the mini cluster to even use the reporter?